FordLabs/PeopleMover
# Add color dropdown to roles modal

## Issue
Resolves #424

## What was done
- [x] Add color dropdown to roles modal

## How to test
1. Ensure that the color dropdown works properly on the roles modal
2. Ensure that the roles and tags modals work as expected

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
